elm.json

```diff
@@ -12,7 +12,8 @@
             "elm/json": "1.1.3",
             "elm/url": "1.0.0",
             "elm-community/json-extra": "4.3.0",
-            "rtfeldman/elm-css": "18.0.0"
+            "rtfeldman/elm-css": "18.0.0",
+            "the-sett/elm-string-case": "1.0.2"
```

Comment:
We probably won't need to manipulate that string afterall - but if we did core String library has mostly what you'd need to do to uppercase & replace dash with space etc. It's counter intuitive when coming from the js ecosystem, but generally better to err on the side of not importing packages to do stuff because they add bloat and usually don't fit your exact use case. A lot of thinking went into providing most utility function needed in the Core libraries. There are of course times when we don't want to reinvent the wheel, so it's not a blanket rule!

https://package.elm-lang.org/packages/elm/core/latest/String